.github/workflows/ios-custom-build.yml

```
@@ -30,6 +30,17 @@ on:
           - collab
           - synergy
           - mec
+      internal-testers:
+        description: 'Internal Testers Group'
+        required: true
+        default: 'QA-Triple'
```

actionlint (1.7.12)

[error] default value "QA-Triple" of "internal-testers" input is not included in its options "\"Dev-testing\", \"MEC\", \"QA\", \"MOSIP-Collab\", \"MOSIP-Dev-testing\""

(events)